templates/components/password.build.hbs

```diff
@@ -2,7 +2,7 @@
   type="password"
   class="form-control"
   :readonly="autogenerate"
-  autocomplete="off"
+  autocomplete="new-password"
```

Comment:
why `new-password`? If it needs to be a random field name then you can generate it with `Fliplet.guid()`

Reply:
This is the only way it works.

Reply:
@maksim-an-upplabs can you elaborate? post relevant documentation to support the argument?

Reply:
@MaksymShokin can you elaborate? post relevant documentation to support the argument?

Reply:
@squallstar 
After I tried other methods I found this article https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion.
Below it has section called "Preventing autofilling with autocomplete="new-password"."